regexes/device/mobiles.yml

```diff
@@ -40506,6 +40579,30 @@ ANBERNIC:
   device: 'console'
   model: '$1'
 
+# Aocwei
+Aocwei:
+  regex: '(?:X900_US)(?:[);/ ]|$)'
+  device: 'tablet'
+  model: 'X900'
+
+# SMUX or SMUXBOX
+SMUX:
+  regex: 'SMUX ?BOX A1(?:[);/ ]|$)'
+  device: 'tv'
+  model: 'A1'
+
+# AAUW (aauwtablet.com)
+AAUW:
+  regex: 'Android 13(?:[\d.]*);(?: [\w-]+;)? (?:M50)( Build|[;)])'
+  device: 'tablet'
+  model: 'M50'
+
+# Rocket Tablet Computer
+Rocket:
```

Comment:
is it worth specifying the full name as on the cover of the device?
<details>
<summary>Preview image</summary>

![image](https://github.com/matomo-org/device-detector/assets/1337066/5095b730-0e43-4938-bdad-ad1485e1d2f1)

</details>

Reply:
I think it is. We already have some full names and also will avoid confusion for further detections.